tests/test_basic.py

```diff
@@ -56,10 +57,11 @@ def save(self, *args, **kwargs):
 
 class AlbumModelViewset(viewsets.ModelViewSet):
     authentication_classes = [TokenAuthentication]
+    permission_classes = [IsAuthenticatedOrReadOnly]
     serializer_class = AlbumSerializer
     queryset = Album.objects.none()
 
-    @action(detail=True, methods=['POST'], serializer_class=LikeSerializer)
+    @action(detail=True, methods=['POST'], serializer_class=LikeSerializer, permission_classes=[IsAuthenticated])
```

Comment:
setting `IsAuthenticated` here is superfluous, as this POST is already covered by the `IsAuthenticatedOrReadOnly`

Reply:
Nice catch! Addressed in a361a31.